src/main/java/com/google/alpollo/Song.java

```diff
@@ -45,4 +33,10 @@ public String getName() {
   public String getAlbum() {
     return album;
   }
+
+  /** Calculates and returns song id by each song. */
+  static String id(Song song) {
+    // TODO: add hashcode instead of concatenation
+    return song.artist + song.name + song.album;
+  }
```

Comment:
Why is this a static method that receives a song instead of an instance method with no parameters that returns `this.artist + this.name + this.album`?